### ISSUE TITLE
Run local auth in minshift

### ISSUE THREAD

**Issue by alexeykazakov** [fabric8-services/fabric8-auth#194]
We should add a target in our `make` to build an image from local sources and deploy them to a minishift VM
It can be based on https://github.com/fabric8-services/fabric8-wit/pull/1741 but we need to run Auth in minishift too. Not just our dependencies.

**Comment by pradeepto** [comment]
>We should add a target in our make to build an image from local sources and deploy them to a minishift VM
It can be based on fabric8-services/fabric8-wit#1741 but we need to run Auth in minishift too. Not just our dependencies.

@alexeykazakov auth is already running inside minishift as a part of fabric8-services/fabric8-wit#1741 You can see it [here](https://github.com/fabric8-services/fabric8-wit/pull/1741/files#diff-a9a855877e71a3b2f42bb232be87078c).

cc @pranavgore09 @sbose78

**Comment by pradeepto** [comment]
@alexeykazakov  fabric8-services/fabric8-wit#1741 does everything but for "building and deploying the service being developed". It currently "builds and deploys" all the dependent services on minishift using kedge but runs the "service being developed" on the localhost directly. This needs to be done.

**Comment by alexeykazakov** [comment]
@LalatenduMohanty sure. Thanks for the help.
@pradeepto yeah.. I think we will need both options. We need to be able to run the dependencies on minishift but the auth itself on the localhost as WIT does. It's useful when you don't want to deal with setting up decencies but need some very quick way to build and run Auth from sources during development. And we also need a command to run all in minishift (build an auth image from sources and deploy it to minishift) for testing everything in OS environment.

**Comment by pradeepto** [comment]
@alexeykazakov Agreed. I was just stating what exists now and what needs to be added.
